frontend/src/containers/Event/Event.tsx

```diff
@@ -17,6 +17,9 @@ import Link from 'next/link'
 import { calculateDuration } from '@/utils'
```

Comment:
Can you add screenshots of the UI changes. It makes it easier for reviewer to confirm the expected changes/behaviour